lib/jerboa/format/stun/decode_error.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Jerboa.Format.STUN.DecodeError do
+  @moduledoc """
+  Data structure containing information about decoding failure
+  """
+
+  defstruct [format: "", method: ""]
```

Comment:
It shouldn't be here yet. It is used in upcoming PR. I'll remove it from this one.